packages/fast-components-react-msft/src/background/README.md

```diff
@@ -0,0 +1,44 @@
+# Background
+
+The *background* component is used to create regions of a page that has a background color. The component will coordinate applying the supplied background color to both the created element *and* adjusting the `DesignSystem` for all child components.
+
+#Usage
```

Comment:
nit: usage should have two # because you can only have one h1, also needs a space after # or it will not render correctly.